### ISSUE TITLE
Enabled/Disabled status isn't updated in the axis poll method

### ISSUE THREAD

**Issue by kmpeters** [epics-motor/motorAutomation1#2]
The axis poll method doesn't query/update the enabled/disabled status of an axis.  If axes are enabled from vendor software while the IOC is running, the CNEN field will be out-of-sync.

**Comment by GarrettSullivan-Aerotech** [comment]
@kmpeters Would you like us to look at this issue? The enabled/disabled status of an axis corresponds to the Automation1DriveStatus_Enabled bit in the Automation1DriveStatus status item. The fix would likely be as straightforward as adding the status item to statusConfig_, applying the bit mask to the result, then setting the bit in the motor record.

**Comment by kmpeters** [comment]
@GarrettSullivan-Aerotech sure.  I haven't had a chance to look at it yet.

I try to create github issues as I find them, so that they aren't forgotten.

**Comment by GarrettSullivan-Aerotech** [comment]
I pushed a fix for this issue my new branch. It seemed to be working as expected in my test environment.

**Comment by kmpeters** [comment]
It looks like the ``Automation1AxisStatusItem_ProgramPositionCommand`` was removed in your branch.  I assume that is the theoretical or target position of the controller.  Can the Automation1 platform drive motors in open-loop mode without feedback?  If so, is the removal of the commanded position a bad idea, since the theoretical position is the only position for a motor of that type.

**Comment by GarrettSullivan-Aerotech** [comment]
Based on Automation1's implementation of [setClosedLoop](https://github.com/epics-motor/motorAutomation1/blob/main/automation1App/src/Automation1MotorAxis.cpp#L251-L258), open-loop in Automation1 means that the axis is disabled, and thus the user cannot perform motion on the axis at all.

You are correct that ` Automation1AxisStatusItem_ProgramPositionCommand` is the target position of a controller axis. I had originally used it to determine if there was a following error on the axis, however, I realized that the Automation1 C API has a fault bit for following error. When I made the switch to using the fault bit, I forgot to remove that status item from the query; it is not currently being used.
